### PR TITLE
Add OSN 2019 Open Contest Page

### PR DESCRIPTION
**Pull request by yonasadiel**

### CODE REVIEW

File: open.md [hunk 1]
```diff
@@ -4,4 +4,111 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* **Creative** scoring:
+  * The scoring formula will vary for each problem and will be explicitly stated in problem description.
+  * Example:
+    * Batch: [Membaca, OSN 2012](https://training.ia-toki.org/problemsets/45/problems/224/).
+    * Interactive: [Cat Rumah, OSN 2014](https://training.ia-toki.org/problemsets/39/problems/202/).
+    * Output-only: [Tourism in Palembang, OSN 2016](https://training.ia-toki.org/problemsets/54/problems/259/).
+
+<br>
+
+### Contestant Scoring
```
Comment (fushar) [inline open.md:91]: This does not explain the max score per subtask rule?

**Review by fushar — CHANGES_REQUESTED** [review]

File: open.md (hunk 1 above)
Comment (yonasadiel) [reply open.md:91]: on line 94,
`* A subtask final score is maximum score of its score among all submissions.`

Comment (fushar) [reply open.md:91]: Oh, right

**Review by fushar — APPROVED** [review]

File: open.md [hunk 2]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* Each contestant may submit up to **50 submissions** for each problem.
+* The supported programming languages are **C, C++, and Pascal**.
+* During the competition, each contestant can only see their own scores.
+
+<br>
+
+### Schedule and Technical Details
+
+- The contests will be held at the following time ranges:
+* **Day 0**: [2 July 2019 08:35 - 23:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+OSN+Open+Contest+2019+Day+0&iso=20190701T1535&p1=108&ah=14&am=30) (trial session)
```
Comment (jonathanirvings) [inline open.md]: `s/2 July/1 July`
`s/3 July/2 July`
`s/4 July/3 July`

File: terbuka.md [hunk 3]
```diff
@@ -4,4 +4,50 @@ title: TOKI OSN Open Contest 2019 (Bahasa Indonesia)
...
+- Tidak ada sistem klarifikasi dalam kontes, namun setiap pengumuman pada kontes asli akan diumumkan juga pada kontes terbuka.
+
+<br>
+
+### Kategori
+
+Peserta kontes terbuka terbagi atas 4 (empat) kategori:
+
+- **Alumni**, yaitu siapapun yang pernah mengikuti Pelatnas TOKI.
+- **Siswa**, yaitu seluruh pelajar berkewarganegaraan Indonesia yang statusnya masih siswa SD - SMA di Indonesia, tidak menjadi peserta OSN komputer 2019, dan tidak termasuk kategori Alumni.
```
Comment (jonathanirvings) [inline terbuka.md]: `s/komputer/Informatika`

File: terbuka.md [hunk 4]
```diff
@@ -4,4 +4,50 @@ title: TOKI OSN Open Contest 2019 (Bahasa Indonesia)
...
+- Berhubung soal-soalnya sama dengan OSN Informatika 2019, materi mengacu pada halaman [Silabus Materi](silabus.html).
+
+<br>
+
+### Jadwal dan Teknis Kontes
+
+- Secara umum, teknis sesuai dengan yang tercantum pada halaman [Peraturan Kompetisi](peraturan.html).
+- Kontes dibuka dalam rentang waktu sebagai berikut.
+  * **Hari 0**: 1 Juli 2019 15:35 WIB - 2 Juli 2019 06:05 WIB
+    (sesi latihan, tidak mempengaruhi nilai akhir.)
```
Comment (jonathanirvings) [inline terbuka.md]: nit, opsional: `s/tidak/**tidak**`?

File: open.md [hunk 5]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
 key: open
 ---
 
-To Be Announced
+### Preface
+
+- **TOKI OSN Open Contest 2019** is an open competition mirroring the National Science Olympiad in Informatics 2019 in Indonesia (**OSN Informatika 2019** in Bahasa).
+- OSN Informatika is a national olympiad in informatics for high school students. It is the one of the stages required for Indonesian students to represent Indonesia in IOI (International Olympiad in Informatics)
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/the one/one`

File: open.md [hunk 6]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+- The problemset is the same as the problemset in OSN Informatika 2019, and will be given in Bahasa Indonesia and English.
+- The contest will be held on [TLX (TOKI Online Judge)](https://tlx.toki.id/contests) (you may need to register for a TLX account if you do not have one already).
+- If you have any questions please contact [sc-osn-2019@googlegroups.com](mailto:sc-osn-2019@googlegroups.com).
+
+<br>
+
+### Rules Overview
+
+* There will be **one trial session** and **2 (two) competition days**.
+* Result of trial session does **not** affect scoring.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/trial/the trial`

File: open.md [hunk 7]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
 key: open
 ---
 
-To Be Announced
+### Preface
+
+- **TOKI OSN Open Contest 2019** is an open competition mirroring the National Science Olympiad in Informatics 2019 in Indonesia (**OSN Informatika 2019** in Bahasa).
+- OSN Informatika is a national olympiad in informatics for high school students. It is the one of the stages required for Indonesian students to represent Indonesia in IOI (International Olympiad in Informatics)
+- The problemset is the same as the problemset in OSN Informatika 2019, and will be given in Bahasa Indonesia and English.
```
Comment (jonathanirvings) [inline open.md]: grammarly: remove comma

File: open.md [hunk 8]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+
+<br>
+
+### Problems Types
+
+- There are 3 possible problem types. There may be some problem type that is not given.
+
+- **Batch** problems:
+  * The most common problem type.
+  * Given input, contestants write a program that reads input from **stdin** and writes output to **stdout**.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/Given input/Given an input`

File: open.md [hunk 9]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+  * The program must run within the given time and memory limits.
+  * Example: [Pekanbaru Defense, OSN 2017](https://training.ia-toki.org/problemsets/91/problems/469/)
+
+- **Interactive** problems:
+  * Contestants write a program that interacts with judge's program: judge's program writes to **stdout** and is given as the **stdin** for contestant's program, then contestant's program writes to **stdout** and is given as the **stdin** for judge's program, and the interaction continues until a given goal is reached.
+  * Contestant's program must output answer within the given time and memory limits.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/output answer/output the answer`

File: open.md [hunk 10]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+  * Contestants submit **output** for each input test case.
+  * Contestants **do not have to** write a program; the outputs may be solved manually.
+  * Example: [Tourism in Palembang, OSN 2016](https://training.ia-toki.org/problemsets/54/problems/259/).
+
+<br>
+
+### Problem Scoring
+
+* There are two types of scoring: **standard** and **creative**.
+* Creative problems will be stated explicitly in problem description.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/in problem/in the problem`

File: open.md [hunk 11]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+
+* **Standard** scoring:
+  * For batch and interactive problems:
+    * A problem consists of **multiple subtasks** with various points.
+    * A subtask consists of **multiple test cases** that are grouped into some **test groups**.
+    * A submission gets a subtask's points if it solves **all of its test groups**.
+    * To solve a test group, a submission have to solve **all of its test cases**.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/a submission have/a submission has`

File: open.md [hunk 12]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+    * A problem may have **open subtasks** where the test cases are given to contestants.
+  * For output-only problems:
+    * A problem consists of **multiple test cases** with various points.
+    * A submission gets the test case's points if it produces the correct output.
+
+* **Creative** scoring:
+  * The scoring formula will vary for each problem and will be explicitly stated in problem description.
```
Comment (jonathanirvings) [inline open.md]: grammarly: s/problem description/the problem description`

File: open.md [hunk 13]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+
+<br>
+
+### Contestant Scoring
+
+* For batch and interactive problems:
+  * On creative scoring, for each submission, score of a subtask is the minimum score among all its test cases.
+  * A subtask final score is **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its subtasks final score.
+  For example, if the first submission got 30 on first subtask and 0 on second subtask, and second submission got 0 on first subtask and 40 on second subtask, then the contestant's score on the problem is 70.
```
Comment (jonathanirvings) [inline open.md]: `s/30/30 points`

Comment (jonathanirvings) [inline open.md]: `s/on first subtask/for the first subtask`

Comment (jonathanirvings) [inline open.md]: `s/0/0 points`

Comment (jonathanirvings) [inline open.md]: grammarly: `s/on second subtask/for the second subtask`

Comment (jonathanirvings) [inline open.md]: grammarly: `s/second submission/the second submission`

Comment (jonathanirvings) [inline open.md]: `s/40/40 points`

File: open.md [hunk 14]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* For output-only problems:
+  * A test case final score is the **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its test cases final score.
+* A contestant's total score is sum of **all of problems score** on **both days**.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/sum of/the sum of`

File: open.md [hunk 15]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* Contestants will be ranked by total score (descending).
+* Two contestants with **same** total score will get **same** rank.
```
Comment (jonathanirvings) [inline open.md]: grammarly: `s/**same**/**the same**`

File: open.md [hunk 16]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* Submission submit time **will not affect** contestant rank at all.
```
Comment (jonathanirvings) [inline open.md]: `s/Submission submit time/Submission time`

File: open.md [hunk 17]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+    * Interactive: [Cat Rumah, OSN 2014](https://training.ia-toki.org/problemsets/39/problems/202/).
+    * Output-only: [Tourism in Palembang, OSN 2016](https://training.ia-toki.org/problemsets/54/problems/259/).
+
+<br>
+
+### Contestant Scoring
+
+* For batch and interactive problems:
+  * On creative scoring, for each submission, score of a subtask is the minimum score among all its test cases.
+  * A subtask final score is **maximum of its score** among all submissions.
```
Comment (jonathanirvings) [inline open.md]: `s/is/is the`

File: open.md [hunk 18]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+  * Contestant's score on a problem is the sum of all its subtasks final score.
```
Comment (jonathanirvings) [inline open.md]: `s/on a problem/for a problem`

File: open.md (hunk 13 above)
Comment (jonathanirvings) [inline open.md]: `s/on the problem/for the problem`

File: open.md [hunk 19]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
 key: open
 ---
 
-To Be Announced
+### Preface
+
+- **TOKI OSN Open Contest 2019** is an open competition mirroring the National Science Olympiad in Informatics 2019 in Indonesia (**OSN Informatika 2019** in Bahasa).
+- OSN Informatika is a national olympiad in informatics for high school students. It is the one of the stages required for Indonesian students to represent Indonesia in IOI (International Olympiad in Informatics)
+- The problemset is the same as the problemset in OSN Informatika 2019, and will be given in Bahasa Indonesia and English.
+- The contest will be held on [TLX (TOKI Online Judge)](https://tlx.toki.id/contests) (you may need to register for a TLX account if you do not have one already).
```
Comment (jonathanirvings) [inline open.md]: `already` nya hapus aja deh.

File: open.md [hunk 20]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+- The contests will be held at the following time ranges:
+* **Day 0**: [2 July 2019 08:35 - 23:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+OSN+Open+Contest+2019+Day+0&iso=20190701T1535&p1=108&ah=14&am=30) (trial session)
+* **Day 1**: [3 July 2019 02:35 - 23:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+OSN+Open+Contest+2019+Day+1&iso=20190702T0935&p1=108&ah=20&am=30)
+* **Day 2**: [4 July 2019 02:35 - 23:05 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?msg=TOKI+OSN+Open+Contest+2019+Day+2&iso=20190703T0935&p1=108&ah=20&am=30)
+
+- Each contestant can select **any 5-hour window** within the time range to do the contest for each competition day.
+- All three contests are now available on [TLX](https://tlx.toki.id/contests). Please register in those contests.
+- Each contestant may start the contest any time within the time range, by clicking the timer button.
+- There will be no additional time given if a contestant starts the contest in less than 5 hours before the contest ends.
+- The open contest doesn't support clarifications, but all important announcements from the original contest will be broadcast as well.
```
Comment (jonathanirvings) [inline open.md]: `s/doesn't/does not`

File: open.md [hunk 21]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+
+<br>
+
+### Problems Types
+
+- There are 3 possible problem types. There may be some problem type that is not given.
```
Comment (jonathanirvings) [inline open.md]: `s/There may be some problem type that is not given./Some problem type might not be tested.`

File: open.md [hunk 22]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+  * Contestant's score on a problem is the sum of all its subtasks final score.
+  For example, if the first submission got 30 on first subtask and 0 on second subtask, and second submission got 0 on first subtask and 40 on second subtask, then the contestant's score on the problem is 70.
+* For output-only problems:
+  * A test case final score is the **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its test cases final score.
+* A contestant's total score is sum of **all of problems score** on **both days**.
+* Contestants will be ranked by total score (descending).
+* Two contestants with **same** total score will get **same** rank.
+* Submission submit time **will not affect** contestant rank at all.
+
```
Comment (jonathanirvings) [inline open.md:107]: Add `<br>` here.

File: open.md [hunk 23]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* For batch and interactive problems:
+  * On creative scoring, for each submission, score of a subtask is the minimum score among all its test cases.
+  * A subtask final score is **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its subtasks final score.
+  For example, if the first submission got 30 on first subtask and 0 on second subtask, and second submission got 0 on first subtask and 40 on second subtask, then the contestant's score on the problem is 70.
+* For output-only problems:
+  * A test case final score is the **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its test cases final score.
+* A contestant's total score is sum of **all of problems score** on **both days**.
+* Contestants will be ranked by total score (descending).
```
Comment (jonathanirvings) [inline open.md]: `s/total score/the total score`

File: open.md [hunk 24]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+    * Batch: [Membaca, OSN 2012](https://training.ia-toki.org/problemsets/45/problems/224/).
+    * Interactive: [Cat Rumah, OSN 2014](https://training.ia-toki.org/problemsets/39/problems/202/).
+    * Output-only: [Tourism in Palembang, OSN 2016](https://training.ia-toki.org/problemsets/54/problems/259/).
+
+<br>
+
+### Contestant Scoring
+
+* For batch and interactive problems:
+  * On creative scoring, for each submission, score of a subtask is the minimum score among all its test cases.
```
Comment (jonathanirvings) [inline open.md]: `s/score of a/the score of a`

File: open.md [hunk 25]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+  * A subtask final score is **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its subtasks final score.
+  For example, if the first submission got 30 on first subtask and 0 on second subtask, and second submission got 0 on first subtask and 40 on second subtask, then the contestant's score on the problem is 70.
+* For output-only problems:
+  * A test case final score is the **maximum of its score** among all submissions.
+  * Contestant's score on a problem is the sum of all its test cases final score.
```
Comment (jonathanirvings) [inline open.md]: `s/Contestant's/A contestant's`

File: open.md (hunk 14 above)
Comment (jonathanirvings) [inline open.md]: nit, opsional: `s/both days/both competition days`

File: open.md (hunk 15 above)
Comment (jonathanirvings) [inline open.md]: `s/get/achieve`

File: open.md [hunk 26]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
...
+* A contestant's total score is sum of **all of problems score** on **both days**.
+* Contestants will be ranked by total score (descending).
+* Two contestants with **same** total score will get **same** rank.
+* Submission submit time **will not affect** contestant rank at all.
+
+### Grading System
+
+* Grading system that will be used is [TLX](https://tlx.toki.id).
```
Comment (jonathanirvings) [inline open.md]: `TLX will be used as the grading system`

File: open.md [hunk 27]
```diff
@@ -4,4 +4,112 @@ title: TOKI OSN Open Contest 2019 (English)
 key: open
```
Comment (jonathanirvings) [inline open.md:4]: Tambahin post di `index.html` juga

```
  <div class="pt-card card__title">
    <h3 class="card__title__text">TOKI OSN Open Contest</h3>
    <p class="card__title__right">26 Juni 2019</p>
  </div>
  <div class="pt-card card__content">
    <p>Telah ditambahkan informasi mengenai TOKI OSN Open Contest <a href="terbuka.html">di sini</a>.</p>
    <hr>
    <p>Information regarding TOKI OSN Open Contest has been added <a href="open.html">here</a>.</p>
  </div>
```

**Review by jonathanirvings — CHANGES_REQUESTED** [review]
Sori ada banyak komen, terutama di bagian yang "Contestant Scoring" (ini baru lu tulis taun ini kan ya).

Beberapa grammatical mistake dari yang dicopas taun lalu juga gw rasa critical untuk di fix.

File: index.html [hunk 28]
```diff
@@ -3,6 +3,16 @@
 key: news
 ---
 
+<div class="pt-card card__title">
```
Comment (jonathanirvings) [inline index.html:6]: Wrap ini dan `<div>` bawahnya di `<div>` baru (liat post2 bawahnya sebagai contoh)

File: open.md [hunk 29]
```diff
@@ -91,21 +91,23 @@ key: open
...
 
 * For batch and interactive problems:
-  * On creative scoring, for each submission, score of a subtask is the minimum score among all its test cases.
-  * A subtask final score is **maximum of its score** among all submissions.
-  * Contestant's score on a problem is the sum of all its subtasks final score.
-  For example, if the first submission got 30 on first subtask and 0 on second subtask, and second submission got 0 on first subtask and 40 on second subtask, then the contestant's score on the problem is 70.
+  * On creative scoring, for each submission, the score of a subtask is the minimum score among all its test cases.
+  * A subtask final score is the **maximum of its score** among all submissions.
+  * Contestant's score for a problem is the sum of all its subtasks final score.
+  For example, if the first submission got 30 points on the first subtask and 0 points on the second subtask, and the second submission got 0 points on the first subtask and 40 points on the second subtask, then the contestant's score for the problem is 70 points.
```
Comment (jonathanirvings) [inline open.md:97]: `s/on the/for the` (untuk semua occurence `on the` di line ini)

File: open.md [hunk 30]
```diff
@@ -91,21 +91,23 @@ key: open
...
   * A test case final score is the **maximum of its score** among all submissions.
-  * Contestant's score on a problem is the sum of all its test cases final score.
-* A contestant's total score is sum of **all of problems score** on **both days**.
-* Contestants will be ranked by total score (descending).
-* Two contestants with **same** total score will get **same** rank.
-* Submission submit time **will not affect** contestant rank at all.
+  * A contestant's score on a problem is the sum of all its test cases final score.
+* A contestant's total score is the sum of **all of problems score** on **both competition days**.
+* Contestants will be ranked by the total score (descending).
+* Two contestants with **same** total score will achieve **the same** rank.
```
Comment (jonathanirvings) [inline open.md:103]: yang kiri juga: `s/**same**/**the same**`

**Review by jonathanirvings — APPROVED** [review]
Thanks. 3 more nits, feel free buat langsung merge setelah fix ini.

**Pull request merged**